js/main.js

```diff
@@ -1,17 +1,27 @@
+// Mobile Menu
 const div = document.createElement('div');
 const hamBttn = document.querySelector('.menu');
-const mainpage = document.querySelector('.main-page');
 const header = document.querySelector('header');
+const mainpage = document.querySelector('.main-page');
 const worksSection = document.querySelector('.works-section');
 const aboutMe = document.querySelector('.about');
 const lastPage = document.querySelector('.last-page');
 const blurElements = [mainpage, header, worksSection, aboutMe, lastPage];
 
+// Mobile Menu
 function UnBlurSections() {
   for (let i = 0; i < blurElements.length; i += 1) {
     blurElements[i].classList.remove('blur-content');
   }
 }
+
+function BlurSections() {
+  // Add css class to behind sections
+  for (let i = 0; i < blurElements.length; i += 1) {
+    blurElements[i].classList.add('blur-content');
+  }
+}
+
```

Comment:
- [ ] Kindly ensure arrow functions are used instead of the function keyword if you check [Javascript requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)you would see that the use of ES6 is encouraged. Javascript's ES6 brings new syntax and awesome features to make your code more modern and readable. It allows you to write less code and do more. ES6 introduces us to many great features like arrow functions. The arrow function is really awesome and makes your code more readable, more structured, and look modern. Take a minute and also go through this [article](https://zendev.com/2018/10/01/javascript-arrow-functions-how-why-when.html) on why ES6 arrow functions are so awesome. Then fix this 😄